nastaveni/nastaveni.php

```diff
@@ -86,7 +86,8 @@
 
 define('KURZ_EURO',26.00);            // kurz kč:euro
 define('UCET_CZ','2800035147/2010');  // číslo účtu pro platby v CZK - v statických stránkách není
-define('UCET_SK','2800035147/8330');  // číslo účtu pro platby v EUR - v statických stránkách není
+define('IBAN','CZ2820100000002800035147'); //mezinárodní číslo účtu
```

Comment:
tady by bylo lepší doplnit mezery stejně jako v zbytku souboru (za čárku za parametr funkce, mezi // a začátek komentáře, platí to i pro další řádek)